package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "akismet-api",
-  "version": "2.1.1",
+  "version": "2.1.2",
```

Comment:
Are you able to leave this at 2.1.1? I think this should be 2.2.0 to be in line with semantic versioning, and I'll need to update the CHANGELOG so I'll likely just do a separate PR which bumps the version before publish :)